pom.xml

```diff
@@ -68,7 +68,7 @@
         <metamx.java-util.version>0.27.0</metamx.java-util.version>
         <apache.curator.version>2.7.0</apache.curator.version>
         <jetty.version>9.2.5.v20141112</jetty.version>
-        <druid.api.version>0.3.6</druid.api.version>
+        <druid.api.version>0.3.7-SNAPSHOT</druid.api.version>
```

Comment:
It will need to be updated more anyways (see dependency note in PR master comment)